docs/admin/node.md

```diff
@@ -59,8 +59,16 @@ conditions mean the node is in sane state:
 ]
 ```
 
-If the Status of the Ready condition
-is Unknown or False for more than five minutes, then all of the Pods on the node are terminated by the Node Controller.
+If the Status of the Ready condition is Unknown or False for more than five minutes, then all of the Pods on the node are scheduled for deletion by the Node Controller.
+In some cases when the node is unreachable, the apiserver is unable to communicate with the kubelet on it. 
```

Comment:
I miss `thus` part in this sentence (but I'm not a native speaker, so feel free to ignore)